src/server/utils/prepare.js

```diff
@@ -10,21 +10,43 @@ var exec = require('child_process').exec,
  * @return {Promise}
  */
 function execCordovaPrepare(projectRoot, platform) {
-    var deferred = Q.defer();
+    return retryAsync(getExecCordovaPrepareImpl(projectRoot, platform), 2);
+}
+
+function getExecCordovaPrepareImpl(projectRoot, platform) {
+    return function () {
+        var deferred = Q.defer();
 
-    log.log('Preparing platform \'' + platform + '\'.');
+        log.log('Preparing platform \'' + platform + '\'.');
 
-    exec('cordova prepare ' + platform, {
-        cwd: projectRoot
-    }, function (err, stdout, stderr) {
-        if (err) {
-            deferred.reject(err || stderr);
+        exec('cordova prepare ' + platform, {
+            cwd: projectRoot
+        }, function (err, stdout, stderr) {
+            if (err || stderr) {
```

Comment:
I vaguely remembered a scenario where the reverse was true - no exception was thrown and we only got info about an error through `stderr` rather than `err`. However, since that change isn't needed for this fix, to be safer I'll pull it until we have definite evidence it is needed (and safe).